Tasks/JavaToolInstallerV0/package.json

```diff
@@ -23,12 +23,16 @@
     },
     "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
     "dependencies": {
-        "@types/node": "^6.0.101",
+        "@types/mocha": "^5.2.7",
+        "@types/node": "^10.17.0",
+        "azure-pipelines-task-lib": "^3.0.6-preview.0",
         "@types/q": "^1.0.7",
         "azp-tasks-az-blobstorage-provider-v2": "1.0.0",
-        "azure-pipelines-task-lib": "2.9.3",
         "azure-pipelines-tasks-azure-arm-rest": "1.0.0",
         "azure-pipelines-tool-lib": "0.12.0",
```

Comment:
a new version of ToolLib is out as well (1.0.0-preview.0)

Reply:
@sdobrodeev let's clarify if we should update this dependency in this PR